OPAL/br/src/main/scala/org/opalj/br/fpcf/properties/Alias.scala

```diff
@@ -0,0 +1,86 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package br
+package fpcf
+package properties
+
+import org.opalj.fpcf.Property
+import org.opalj.fpcf.PropertyKey
+import org.opalj.fpcf.PropertyMetaInformation
+
+sealed trait AliasPropertyMetaInformation extends PropertyMetaInformation {
+    type Self = Alias
+}
+
+/**
+ * Describes the alias properties of the associated entities.
+ *
+ * Alias properties can establish a binding between a pair of entities, including definition sites, formal parameters, method return values, fields, etc.
+ * For more information, see TODO.
+ * The pair may consist of different types of entities, such as a field and a formal parameter.
+ *
+ * An alias property provides information about the relationship of the memory locations of the associated entities.
+ *
+ *  - [[NoAlias]] can be assigned to a pair of entities, iff no execution path exists where both entities refer to the same memory location.
+ *
+ *  - [[MustAlias]] can be assigned to a pair of entities, iff both entities refer to the same memory location in every execution time at all times.
+ * This implies that at every use site of one entity, it can be replaced with the other entity (if it is defined at the current location)
```

Comment:
I like the description here, but maybe it should be expanded a little more to be absolutely clear about the meaning? You may also include a small code example if it helps.